teapots/common/ndk_helper/JNIHelper.h

```diff
@@ -178,6 +270,66 @@ class JNIHelper {
    *
    */
   const char* GetAppName() { return app_name_.c_str(); }
+
+  /*
+   * Retrieves application label
+   *
+   * return: pointer to an app label string
+   *
+   */
+  const char* GetAppLabel() { return app_label_.c_str(); }
```

Comment:
changed it, nevertheless, to const char* GetAppLabel() const {return....}  does not feel there is any value added though: it is inline function and code is right there in the header: clients now it right away it does not change anything inside the class